java/console/build.gradle.kts

```diff
@@ -0,0 +1,27 @@
+/*
+ * This file was generated by the Gradle 'init' task.
+ *
+ * This is a general purpose Gradle build.
+ * Learn more about Gradle by exploring our Samples at https://docs.gradle.org/8.11.1/samples
+ */
+
+plugins {
+    `java`
```

Comment:
I think you are safe to lose the apostrophes here

Reply:
Thanks! I have no clue why copilot did that, but you're very right that the backticks are superfluous. Removing!